app/views/layouts/.fuse_hidden00002b640000001b

```diff
@@ -0,0 +1,24 @@
+doctype html
```

Comment:
I'm not sure that this file have to be pushed to remote. I guess, it's some temporary file. please delete it and google why it's created.